{
	"id": "CVE-2020-4651",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2020-4651",
				"description": "IBM Maximo Spatial Asset Management 7.6.0.3, 7.6.0.4, 7.6.0.5, and 7.6.1.0 is vulnerable to cross-site request forgery which could allow an attacker to execute malicious and unauthorized actions transmitted from a user that the website trusts. IBM X-Force ID: 186024.",
				"severity": [
					{
						"type": "cvss_v2",
						"source": "nvd@nist.gov",
						"cvss_v2": {
							"vector": "AV:A/AC:M/Au:N/C:N/I:P/A:N",
							"base_score": 2.9,
							"nvd_base_severity": "LOW",
							"temporal_score": 2.9,
							"nvd_temporal_severity": "LOW",
							"environmental_score": 2.9,
							"nvd_environmental_severity": "LOW"
						}
					},
					{
						"type": "cvss_v31",
						"source": "nvd@nist.gov",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:A/AC:H/PR:N/UI:R/S:U/C:N/I:H/A:N",
							"base_score": 4.8,
							"base_severity": "MEDIUM",
							"temporal_score": 4.8,
							"temporal_severity": "MEDIUM",
							"environmental_score": 4.8,
							"environmental_severity": "MEDIUM"
						}
					},
					{
						"type": "cvss_v30",
						"source": "psirt@us.ibm.com",
						"cvss_v30": {
							"vector": "CVSS:3.0/AV:A/AC:H/PR:N/UI:R/S:U/C:N/I:H/A:N",
							"base_score": 4.8,
							"base_severity": "MEDIUM",
							"temporal_score": 4.8,
							"temporal_severity": "MEDIUM",
							"environmental_score": 4.8,
							"environmental_severity": "MEDIUM"
						}
					}
				],
				"cwe": [
					{
						"source": "nvd@nist.gov",
						"cwe": [
							"CWE-352"
						]
					}
				],
				"references": [
					{
						"source": "af854a3a-2127-422b-91ae-364da2661108",
						"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/186024"
					},
					{
						"source": "af854a3a-2127-422b-91ae-364da2661108",
						"url": "https://www.ibm.com/support/pages/node/6361767"
					},
					{
						"source": "nvd.nist.gov",
						"url": "https://nvd.nist.gov/vuln/detail/CVE-2020-4651"
					},
					{
						"source": "psirt@us.ibm.com",
						"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/186024"
					},
					{
						"source": "psirt@us.ibm.com",
						"url": "https://www.ibm.com/support/pages/node/6361767"
					}
				],
				"published": "2020-11-09T21:15:12.917Z",
				"modified": "2024-11-21T05:33:02.983Z"
			},
			"segments": [
				{
					"ecosystem": "cpe"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "cpe",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterias": [
							{
								"operator": "OR",
								"criterias": [
									{
										"operator": "OR",
										"criterias": [
											{
												"operator": "OR",
												"criterions": [
													{
														"type": "version",
														"version": {
															"vulnerable": true,
															"fix_status": {
																"class": "unknown"
															},
															"package": {
																"type": "cpe",
																"cpe": "cpe:2.3:a:ibm:maximo_spatial_asset_management:7.6.0.3:*:*:*:*:*:*:*"
															}
														}
													}
												]
											},
											{
												"operator": "OR",
												"criterions": [
													{
														"type": "version",
														"version": {
															"vulnerable": true,
															"fix_status": {
																"class": "unknown"
															},
															"package": {
																"type": "cpe",
																"cpe": "cpe:2.3:a:ibm:maximo_spatial_asset_management:7.6.0.4:*:*:*:*:*:*:*"
															}
														}
													}
												]
											},
											{
												"operator": "OR",
												"criterions": [
													{
														"type": "version",
														"version": {
															"vulnerable": true,
															"fix_status": {
																"class": "unknown"
															},
															"package": {
																"type": "cpe",
																"cpe": "cpe:2.3:a:ibm:maximo_spatial_asset_management:7.6.0.5:*:*:*:*:*:*:*"
															}
														}
													}
												]
											},
											{
												"operator": "OR",
												"criterions": [
													{
														"type": "version",
														"version": {
															"vulnerable": true,
															"fix_status": {
																"class": "unknown"
															},
															"package": {
																"type": "cpe",
																"cpe": "cpe:2.3:a:ibm:maximo_spatial_asset_management:7.6.1.0:*:*:*:*:*:*:*"
															}
														}
													}
												]
											}
										]
									}
								]
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "nvd-api-cve",
		"raws": [
			"vuls-data-raw-nvd-api-cve/2020/CVE-2020-4651.json"
		]
	}
}
